test/functional/feature_llmq_chainlocks.py

```diff
@@ -187,6 +187,45 @@ def run_test(self):
         self.reconnect_isolated_node(0, 1)
         self.wait_for_chainlocked_block(self.nodes[0], self.nodes[0].getbestblockhash(), timeout=30)
 
+        self.log.info("Add a new node and let it sync")
+        added_mn_info_0 = self.dynamically_add_masternode(evo=False)
+        has_chainlock_info = False
+        try:
+            self.log.info(f'{self.nodes[added_mn_info_0.idx].getbestchainlock()}')
+            has_chainlock_info = True
+        except:
+            self.log.info("getbestchainlock() failed on the new node as expected")
+        assert not has_chainlock_info
+        self.log.info("Test that new node can mine without Chainlock info")
+        tip_0 = self.nodes[0].getblock(self.nodes[0].getbestblockhash(), 2)
+        self.nodes[added_mn_info_0.idx].generate(1)
+        self.sync_blocks(self.nodes)
+        tip_1 = self.nodes[0].getblock(self.nodes[0].getbestblockhash(), 2)
+        assert_equal(tip_1['cbTx']['bestCLSignature'], tip_0['cbTx']['bestCLSignature'])
+        assert_equal(tip_1['cbTx']['bestCLHeightDiff'], tip_0['cbTx']['bestCLHeightDiff'] + 1)
+
+        self.log.info("Disable Chainlock")
+        self.nodes[0].sporkupdate("SPORK_19_CHAINLOCKS_ENABLED", 4070908800)
+        self.wait_for_sporks_same()
+
+        self.log.info("Add a new node and let it sync")
+        added_mn_info_1 = self.dynamically_add_masternode(evo=False)
+        has_chainlock_info = False
+        try:
+            self.log.info(f'{self.nodes[added_mn_info_1.idx].getbestchainlock()}')
+            has_chainlock_info = True
+        except:
+            self.log.info("getbestchainlock() failed on the new node as expected")
```

Comment:
can `assert_raises_rpc_error` be used here?
Such as 
```suggestion
assert_raises_rpc_error(-32603, "Unable to find any chainlock", self.nodes[added_mn_info_1.idx].getbestchainlock)
````

same for try-except above.
+need to add import `assert_raises_rpc_error`